### PR TITLE
Move flow names to config

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: configurations/pipeline_config.json [hunk 1]
```diff
@@ -15,6 +15,21 @@
     "ac214095-0b42-43d1-9e80-fddf85f4d3c5",
     "1519d5c9-741f-4533-b374-c1cc8d1aa931"
     ],
+
```
Comment (as2388) [inline configurations/pipeline_config.json]:
```suggestion
```

File: configurations/pipeline_config.json [hunk 2]
```diff
@@ -15,6 +15,21 @@
...
+    "ActivationFlowNames" : [
+         "icraf_s01_e01_activation",
+         "icraf_s01_e02_activation",
+         "icraf_s01_e03_activation",
+         "icraf_s01_e04_activation",
+         "icraf_s01_e05_activation",
+         "icraf_s01_e06_activation",
+         "icraf_s01_e07_activation"
+    ],
+
```
Comment (as2388) [inline configurations/pipeline_config.json]:
```suggestion
```

File: run_scripts/run_pipeline.sh [hunk 3]
```diff
@@ -22,9 +22,9 @@ DATA_BACKUPS_DIR=$8
 
 ./1_coda_get.sh "$CODA_PULL_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
 
-./2_fetch_raw_data.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$PIPELINE_CONFIGURATION" "$DATA_ROOT"
+./2_fetch_raw_data.sh "$USER" "$PIPELINE_CONFIGURATION_FILE_PATH" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$DATA_ROOT"
 
-./3_generate_outputs.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$PIPELINE_CONFIGURATION" "$DATA_ROOT"
+./3_generate_outputs.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$DATA_ROOT"
```
Comment (as2388) [inline run_scripts/run_pipeline.sh:27]: Surely you still need the pipeline configuration file path here?

**Review by as2388 — APPROVED** [review]

File: run_scripts/run_pipeline.sh (hunk 3 above)
Comment (IsaackMwenda) [reply run_scripts/run_pipeline.sh:27]: So the pipeline copies the pipeline configuration file as a DockerFile step here https://github.com/AfricasVoices/Project-ICRAF/blob/34873923c4c1091a46d170f7ad5267de3c7d7180/Dockerfile#L33 and passes the path in the docker-run.sh here https://github.com/AfricasVoices/Project-ICRAF/blob/34873923c4c1091a46d170f7ad5267de3c7d7180/docker-run.sh#L60. Therefore the argument was never needed in this script. My thinking here is to remove this and standardise this as is on other pipelines but in a different PR. @as2388  what your thoughts?

Comment (as2388) [reply run_scripts/run_pipeline.sh:27]: Ah, I see. Yes, the config should be moved out of the Dockerfile and into an argument, but not in this PR.

Comment (IsaackMwenda) [reply run_scripts/run_pipeline.sh:27]: Great merging!

**Pull request merged**